### ISSUE TITLE
Adding external collaborator ORCID

### ISSUE THREAD

**Issue by hrzepa** [ICHPC/hpc-repo#7]
The labels/actions are confusing.  Thus  ORCID1 has the value  "NAME".   Someone less familiar might just type the actual name of the collaborator, whereas I presume the value is supposed to be the 16-digit ORCID?  Should name be  instead labelled  Identifier?

Or is it supposed to be populated by first conducting a search in the box below, and copy/pasting the appropriate ORCID in? 

When a valid ORCID is supplied, the "Name" field expands into two, the numerical identifier AND the name.

**Comment by mj-harvey** [comment]
Yes, the UX is a bit broken right now. The intention is that the user pastes in the ORCID, which the repo will automatically resolve to the owner's name.
